sequencer/src/block2/tables.rs

```diff
@@ -0,0 +1,260 @@
+use crate::block2::entry::TxTableEntry;
+use crate::block2::payload::{NameSpaceTable, Payload, TableWordTraits};
+use crate::{BlockBuildingSnafu, Error, VmId};
+use derivative::Derivative;
+use serde::{Deserialize, Serialize};
+use snafu::OptionExt;
+use std::marker::PhantomData;
+use std::mem::size_of;
+use std::ops::Range;
+
+#[derive(Clone, Debug, Derivative, Deserialize, Eq, Serialize)]
+#[derivative(Hash, PartialEq)]
+
+// TODO (Philippe) make it private?
```

Comment:
Yes. See other PR review comment.

Reply:
`NamespaceInfo` struct moved to `payload.rs` and made private in 7e1a1ee.